src/components/Calculator.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import Header from './Header';
+import Display from './Display';
+import ButtonContainer from './ButtonContainer';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const [calcObject, setCalcObject] = useState({
+    num1: null,
+    num2: null,
+    displayValue: null,
+    operator: null,
+  });
+
+  const handleButtonClick = (input) => {
+    const newCalcObject = calculate(input, calcObject);
+    setCalcObject(newCalcObject);
+  };
+
+  return (
+    <>
```

Comment:
There's no need for fragment tags when there's only one child element/component, let's remove this please